ops/services/app_functions/report_stream_batched_publisher/infra/main.tf

```diff
@@ -2,6 +2,7 @@ locals {
   simple_report_callback_url = "https://${var.environment == "prod" ? "www" : var.environment}.simplereport.gov/api/reportstream/callback"
   resource_group_name        = "${var.resource_group_name_prefix}${var.environment}"
   report_stream_url          = "https://${var.environment == "prod" ? "" : "staging."}prime.cdc.gov/api/reports?option=SkipInvalidItems&verbose=true"
```

Comment:
I think we should remove the `&verbose=true` part of the URL here

Reply:
Done, new commit pushed